adafruit_drv2605.py

```diff
@@ -327,4 +328,4 @@ def __iter__(self):
 
     def __repr__(self):
         """Return a string representation of all slot's effects."""
-        return repr([effect for effect in self])
+        return repr(list(effect for effect in self))
```

Comment:
@dherrada Why was that change required? Using `list()` isn't as good performance-wise, and isn't really as Pythonic.

Reply:
Black or lint has a validator for unnecessary list comprehensions like this. Comprehensions should only be used when mutating each element (aka more than just `effect` at the start.)

The simplest way to change the `self` object to a list is `list(self)`.

```suggestion
        return repr(list(self))
```